tests/helpers.bash

```diff
@@ -559,3 +565,125 @@ function stop_git_daemon() {
     rm -f ${TESTDIR}/git-daemon/pid
   fi
 }
+
+# Bring up a registry server using buildah with vfs and chroot as a cheap
+# substitute for podman, accessible only to user $1 using password $2 on the
+# local system at a dynamically-allocated port.
+# Requires openssl.
+# If a user name and password are supplied as the two parameters, htpasswd
+# should be available so that they can be hashed.
+# Sets REGISTRY_PID, REGISTRY_PORT (to append to "localhost:"), and
+# REGISTRY_DIR (where the CA cert can be found) on success.
+function start_registry() {
+  local REGISTRY_IMAGE=docker.io/library/registry:2
```

Comment:
I think we may call this a fair number of times during the tests and I wonder if we'll hit the rate limits.  Would it make sense to have our own registry image tucked away on quay.io somewhere?  Do they have one for Podman already?

Reply:
I'm not against that, but I have no idea if we have a plan in place to keep any of those up to date.  The start_registry function uses _prefetch, so it _should_ be pulled, at most, once in a given CI job.

Reply:
Consider using `quay.io/libpod/registry:2.6`. I recall that an update to the one on Docker Hub once broke gating.

Reply:
Using `2.7` now.  `2.6` and `2.7` are not multi-arch, and `2.8` isn't in that repository.  How/when do they get updated?

Reply:
@edsantiago, feel free to ping me next time. Still enjoying fast French fibre :)

Reply:
Done:
```console
$ buildah manifest inspect quay.io/libpod/registry:2.8 | jq '.manifests[].platform.architecture'
"amd64"
"arm"
"arm"
"arm64"
"ppc64le"
"s390x"
```